index.html

```diff
@@ -45,7 +45,7 @@ <h2 class="subtitle bold"><q>Hello! Sharing world</q></h2>
                 <p class="description">
                     A joyful celebration in the value of openess and sharing,
                     creating a positive change with people from over 10 countries
-                    is taking place in July, in Benin.
+                    is taking place in July, in Cotonou, Benin.
                 </p>
                 <div class="hero-event">
```

Comment:
- [ ] Kindly it will be better if you fix the responsiveness at the index page as there is a fault in the first section I think it is needed some height to make it like the Behnace in the responsive,so let's try to fix the responsive at this part to let the user be able to see the project from all sizes of screen 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/lily4178993/leaders-week/assets/83360636/edad4210-1c71-4951-840d-313ac5f60539)    |    ![image](https://github.com/lily4178993/leaders-week/assets/83360636/1b45f445-65c3-491c-8918-8910c4ce3c58)  |